src/Components/Facility/DoctorsCountCard.tsx

```diff
@@ -50,10 +50,65 @@ const DoctorsCountCard = (props: DoctorsCountProps) => {
 
   const area = specialization ? specialization.text : "Unknown";
 
+  useEffect(() => {
+    if (area === "General Medicine") {
+      setColor("#D79B00");
+    } else if (area === "Critical Care") {
+      setColor("#C81E1E");
+    } else if (area === "Paediatrics") {
+      setColor("#453C52");
+    } else if (area === "Other Speciality") {
+      setColor("#03543F");
+    } else if (area === "Pulmonology") {
+      setColor("#000080");
+    }
+  }, [color, area]);
```

Comment:
if these color's are not there in our existing tailwind color palette, add it to tailwind config similar to how patient colors were added. 
![image](https://user-images.githubusercontent.com/25143503/199669809-15aec8fe-4a64-4c58-b0dc-529cb1316ce5.png)